lib/utils/correlation.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+const UUID = require('node-libuuid');
+
+/**
+ * Create a correlation id
+ * @return {String}
+ */
+exports.create = function create() {
+  const id = UUID.v4();
+
+  Log.log('INFO', `Created correlation id ${id}`);
```

Comment:
Do we want to log this at the `LeaseManager` instead so we get a bit more context around which manager is creating the correlation ID?